front/src/modules/ui/dropdown/components/DropdownButton.tsx

```diff
@@ -65,6 +67,15 @@ export const DropdownButton = ({
 
   return (
     <div ref={containerRef}>
+      {dropdownHotkeyScope?.scope && onClose ? (
+        <HotkeyEffect
```

Comment:
HotkeyEffect is here to allow developers to specify a hotkey to toggle the dropdown.

In our case we want every dropdown to close on "esc" key, so we can directly use `useScopedHotkey` in DropdownButton.